tracer/src/Datadog.Trace/Ci/Agent/MessagePack/EventMessagePackFormatter.cs

```diff
@@ -2,34 +2,38 @@
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
+#nullable enable
+#pragma warning disable SA1402
```

Comment:
nit: what's this warning? It's handy to put a comment to the right about what's being disabled 🙂

Reply:
done in [3d98a45](https://github.com/DataDog/dd-trace-dotnet/pull/6374/commits/3d98a45a5bebdc0c7de9990b062d3e8eb506713d)